### PR TITLE
improvement: refactor code, upgrade packages, update webpack and karma config, fix travis config

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
connected with: https://trello.com/c/Y46On2HB/470-angular-phrase-and-angular-phrase-demo-refactor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,18 @@
 
 [![Build Status](https://travis-ci.org/phrase/angular-phrase.png)](https://travis-ci.org/phrase/angular-phrase)
 
-angular-phrase is an addon for [angular-translate](https://github.com/angular-translate/angular-translate) that lets you connect localized AngularJS applications to the Phrase In-Context Editor.
+angular-phrase is an addon for [angular-translate](https://github.com/angular-translate/angular-translate) that lets you connect localized AngularJS applications to the PhraseApp In-Context Editor.
```
Comment (theSoenke) [inline README.md]: This should stay as is. Maybe a rebase issue?

File: README.md [hunk 2]
```diff
@@ -2,19 +2,18 @@
...
 To use angular-phrase with your application you have to:
 
 * Sign up for a Phrase account: [https://phrase.com/en/signup](https://phrase.com/en/signup)
 * Use the excellent [angular-translate](https://github.com/angular-translate/angular-translate) module by Pascal Precht for localization in your AngularJS app
-* Have jQuery included
 
 ## Demo
 
-E-Mail: demo@phrase.com
+E-Mail: demo@phraseapp.com
```
Comment (theSoenke) [inline README.md]: This change should also not be there

File: package.json [hunk 3]
```diff
@@ -1,43 +1,42 @@
 {
   "name": "angular-phrase",
-  "version": "2.1.3",
+  "version": "2.1.4",
   "description": "Use angular-translate with the powerful PhraseApp In-Context-Editor for easy translation management in AngularJS applications",
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/phrase/angular-phrase.git"
+    "url": "git+ssh://git@github.com/phrase/angular-phrase.git##upgrade-version-and-refactor"
```
Comment (theSoenke) [inline package.json]: Why add "#upgrade-version-and-refactor"?

File: src/Data_utils.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export default class DataUtils {
```
Comment (theSoenke) [inline src/Data_utils.ts]: Capital letter for the file name seems a bit unusual to me

File: README.md (hunk 1 above)
Comment (Szymon-dziewonski) [reply README.md]: Yes it is as it was, I changed it at first then realized that it shouldn't be changed

File: package.json (hunk 3 above)
Comment (Szymon-dziewonski) [reply package.json]: I used it on angular-phrase-demo, downloaded this branch for testing purposes

File: README.md (hunk 2 above)
Comment (Szymon-dziewonski) [reply README.md]: there is no demo@phrase.com account which you can you for demo

**Pull request merged**